hosting/technical/catalog_instructions.qmd

```diff
@@ -0,0 +1,41 @@
+# Creating Intake catalogs
+
+There are two versions of [intake](https://intake.readthedocs.io/en/latest/). Here, we focus on intake version 1. Version 2 has advanced features, but a [(partially) incompatible syntax](https://intake.readthedocs.io/en/latest/scope2.html#relationship-to-v1).
+
+## Simple entry
+
+As a minimal catalog, this yaml file would work:
```

Comment:
Might want to note what 'ngc3028' actually means (I think it is a run name?). Also when would chunks not be null and consolidated not be true?